src/VideoManager/VideoManager.h

```diff
@@ -28,6 +28,13 @@ class VideoSettings;
 class Vehicle;
 class Joystick;
 
+enum {
+    TARGET_UNUSED=0,
+    TARGET_LOCK,
+    TARGET_ATTACK,
```

Comment:
Let's use more neutral word instead of attack: follow, start etc.

Reply:
done. updated in all methods and variables - attack -> follow